From 3d296be32c7b7ceb4ee775656680413bfed1cd92 Mon Sep 17 00:00:00 2001
From: Yuwei Chen <yuwei.chen@intel.com>
Date: Mon, 24 Aug 2020 13:00:32 +0800
Subject: [PATCH] Test case for Library - CpuSleep.nasm file

Description: Delete the default setting in the MdePkg/Library/BaseCpuLib/X64/CpuSleep.nasm

Repo: edk2

Signed-off-by: Yuwei Chen <yuwei.chen@intel.com>
---
 MdePkg/Library/BaseCpuLib/X64/CpuSleep.nasm | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/MdePkg/Library/BaseCpuLib/X64/CpuSleep.nasm b/MdePkg/Library/BaseCpuLib/X64/CpuSleep.nasm
index 6a0ddaf2bd..be89a4f123 100644
--- a/MdePkg/Library/BaseCpuLib/X64/CpuSleep.nasm
+++ b/MdePkg/Library/BaseCpuLib/X64/CpuSleep.nasm
@@ -27,5 +27,5 @@
 global ASM_PFX(CpuSleep)
 ASM_PFX(CpuSleep):
     hlt
-    ret
+;    ret
 
-- 
2.27.0.windows.1

